include/rocksdb/advanced_options.h

```diff
@@ -281,6 +281,18 @@ struct AdvancedColumnFamilyOptions {
   // Dynamically changeable through SetOptions() API
   double memtable_prefix_bloom_size_ratio = 0.0;
 
+  // Enable whole key bloom filter in memtable. Note this will only take effect
+  // if memtable_prefix_bloom_size_ratio is not 0. Whole key filter can be
+  // useful when prefix bloom filter's false positive rate is too high, e.g.
```

Comment:
Do we want to mention prefix here at all?
Prefix iterating is much less infrequently used feature.